src/views/partials/board-settings.handlebars

```diff
@@ -0,0 +1,42 @@
+<a href="/board/{{board._id}}">Back</a>
+<div id="boardSettings">
+    <!-- Form for changing the board settings -->
+    <form action="/board/settings/{{board._id}}" method="POST">
+        <div class="boardNameDiv">
+            <label for="boardName">Name: </label>
+            <input id="boardName" type="text" placeholder="Board Name" value="{{board.boardName}}"/>
+        </div>
+        <div class="colorDiv">
+            <label for="positionValue">Color: </label>
+            <input id="positionValue" type="text" placeholder="Color" value="{{board.color}}"/>
+        </div>
+        <div class="descriptionDiv">
+            <label for="positionValue">Description: </label>
+            <input id="positionValue" type="text" placeholder="Description" value="{{board.description}}"/>
+        </div>
+        <div class="submitButtonDiv">
+            <button type="submit">Submit</button>
+        </div>
+    </form>
+    <!-- form for deleting a board -->
+    <form action="/board" method="POST">
```

Comment:
like marco said in the resolved comment for this, doesn't need to be a form, although i dont care if it is. 
But the link this gets sent to should be a POST to "/board/remove/:id" where id is the boardId. That is the route that will remove the user from the board (and delete the board if there is only one user left on it).

Reply:
I think that the form is fine, you can either do client side js which I don't like, or have a simple form. I'm going with the form.

Reply:
cool, form is good. just make sure the link is right